templates/openstack/install.html

```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
+ceph-osd/2 active idle 3 172.16.7.162 Unit is ready (1 OSD)
+ceph-radosgw/0* active idle 0/lxd/0 172.16.7.166 80/tcp Unit is ready
+cinder/0* active idle 1/lxd/2 172.16.7.168 8776/tcp Unit is ready
+cinder-ceph/0* active idle 172.16.7.168 Unit is ready
+glance/0* active idle 2/lxd/1 172.16.7.167 9292/tcp Unit is ready
+keystone/0* active idle 3/lxd/2 172.16.7.169 5000/tcp Unit is ready
+mysql/0* active idle 0/lxd/2 172.16.7.170 3306/tcp Unit is ready
+neutron-api/0* active idle 1/lxd/2 172.16.7.171 9696/tcp Unit is ready
+neutron-gateway/0* active idle 0 172.16.7.159 Unit is ready
+ntp/0* active idle 172.16.7.159 123/udp chrony: Ready
+nova-cloud-controller/0* active idle 2/lxd/2 172.16.7.172 8774/tcp,8775/tcp
+Unit is ready
+nova-compute/0* active idle 1 172.16.7.160 Unit is ready
+neutron-openvswitch/2 active idle 172.16.7.160 Unit is ready
+ntp/3 active idle 172.16.7.160 123/udp chrony: Ready
+nova-compute/1 active idle 2 172.16.7.161 Unit is ready
+neutron-openvswitch/1 active idle 172.16.7.161 Unit is ready
+ntp/2 active idle 172.16.7.161 123/udp chrony: Ready
+nova-compute/2 active idle 3 172.16.7.162 Unit is ready
+neutron-openvswitch/0* active idle 172.16.7.162 Unit is ready
+ntp/1 active idle 172.16.7.162 123/udp chrony: Ready
+openstack-dashboard/0* active idle 3/lxd/2 172.16.7.174 80/tcp,443/tcp Unit is
+ready
+placement/0* active idle 2/lxd/3 172.16.7.175 8778/tcp Unit is ready
+rabbitmq-server/0* active idle 0/lxd/2 172.16.7.176 5672/tcp Unit is ready
+
+Machine State DNS Inst id Series AZ Message
+0 started 172.16.7.159 openstack1 bionic default Deployed
+0/lxd/0 started 172.16.7.166 juju-592de2-0-lxd-3 bionic default Container
+started
+0/lxd/1 started 172.16.7.170 juju-592de2-0-lxd-4 bionic default Container
+started
+0/lxd/2 started 172.16.7.176 juju-592de2-0-lxd-5 bionic default Container
+started
+1 started 172.16.7.160 openstack2 bionic default Deployed
+1/lxd/0 started 172.16.7.164 juju-592de2-1-lxd-3 bionic default Container
+started
+1/lxd/1 started 172.16.7.168 juju-592de2-1-lxd-4 bionic default Container
+started
+1/lxd/2 started 172.16.7.171 juju-592de2-1-lxd-5 bionic default Container
+started
+2 started 172.16.7.161 openstack3 bionic default Deployed
+2/lxd/0 started 172.16.7.163 juju-592de2-2-lxd-4 bionic default Container
+started
+2/lxd/1 started 172.16.7.167 juju-592de2-2-lxd-5 bionic default Container
+started
+2/lxd/2 started 172.16.7.172 juju-592de2-2-lxd-6 bionic default Container
+started
+2/lxd/3 started 172.16.7.175 juju-592de2-2-lxd-7 bionic default Container
+started
+3 started 172.16.7.162 openstack4 bionic default Deployed
+3/lxd/0 started 172.16.7.165 juju-592de2-3-lxd-3 bionic default Container
+started
+3/lxd/1 started 172.16.7.169 juju-592de2-3-lxd-4 bionic default Container
+started
+3/lxd/2 started 172.16.7.174 juju-592de2-3-lxd-5 bionic default Container
+started
+          </code></pre>
+          <p>This is what you should be able to see in the &lsquo;Machines&rsquo; tab in MAAS:</p>
```

Comment:
Single vs double quotes inconsistency:
```suggestion
          <p>This is what you should be able to see in the &ldquo;Machines&ldquo; tab in MAAS:</p>
```